index.js

```diff
@@ -2,32 +2,79 @@
 const {promisify} = require('util');
 const dgram = require('dgram');
 const dns = require('dns-socket');
-const got = require('got');
+const {get: got, CancelError} = require('got');
```

Comment:
```suggestion
const {got, CancelError} = require('got');
```

`get` is the default.